dataprofiler/profilers/unstructured_text_profile.py

```diff
@@ -92,11 +95,22 @@ def __init__(self, name, options=None):
             'hereupon', 'done', 'against', 'get', 'behind', 'several', 'anyone',
             'seeming', "shoulve"}
 
+        if options and options.stop_words is not None:
+            self._stop_words = options.stop_words
 
-        self.__calculations = {
-            "vocab": TextProfiler._update_vocab,
-            "words": TextProfiler._update_words,
-        }
+        self.__calculations = {}
+        if options:
+            if options.vocab.is_enabled:
+                self.__calculations.update(
+                    {"vocab": TextProfiler._update_vocab})
+            if options.words.is_enabled:
+                self.__calculations.update(
+                    {"words": TextProfiler._update_words})
+        else:
+            self.__calculations = {
+                "vocab": TextProfiler._update_vocab,
+                "words": TextProfiler._update_words,
+            }
```

Comment:
this is covered by `BaseColumnProfiler._filter_properties_w_options(self.__calculations, options)`

Reply:
please revert

Reply:
true, changed it back